code-studio/build-js.js

```diff
@@ -31,6 +36,16 @@ build_commands.execute([
     commonFile: 'code-studio-common',
     shouldMinify: commander.min,
     shouldWatch: commander.watch
+  }),
+  build_commands.browserify({
+    srcPath: SRC_PATH + 'initApp/',
+    buildPath: BUILD_PATH,
+    filenames: [
+      'initApp.js'
```

Comment:
What if you named it `'leveltype_all.js'` and pulled `'leveltype_widget.js'` out of the build step above, leaving it levelbuilder-specific?  Since this file is loaded in the _apps_dependencies partial and is apps-specific, that sort of seems appropriate.